lua/toggleterm/terminal.lua

```diff
@@ -106,13 +106,7 @@ local function setup_buffer_autocommands(term)
       "startinsert!",
     })
   end
-  if conf.persist_size and term:is_split() then
```

Comment:
This seems to me like persist size is no longer being respected and no longer specific to just terminals

Reply:
I forgot to check the value of `persist_size` where window sizes are saved.